Nav BridgeIT/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="pl">
+<head>
+  <title>Home</title>
+  <meta charset="utf-8" >
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
+  <link rel="stylesheet" href="css/style.css" type="text/css"/>
+  <link rel="stylesheet" href="icon/css/fontello.css" type="text/css" />
+</head>
+<body>
+  <nav>
+    <div class="menuIcon"><button class="icon-heart"></button></div>
+    <div class="flexWrapper " id="menu">
+      <div class="topLogo">
+        LOGO
+      </div>
+      <ul class="menuItem flexMenu">
+        <a href="#"><li>Option</li></a>
+        <a href="#"><li>Option</li></a>
+        <a href="#"><li>Option</li></a>
+        <a href="#"><li>Option</li></a>
+        <a href="#"><li>Option</li></a>
+      </ul>
+      <div class="flexBetween">
```

Comment:
flexBetween nie mówi o niczym co się dzieję w środku. Nadaj nazwę odpowiadającom stanowi faktycznemu np. menuButtonBox (oczywiście to tylko przykład ale pokazuje, że od razu wiadomo co znajduje się w tej strukturze)